src/org/zaproxy/zap/extension/pscanrulesAlpha/XPoweredByHeaderInfoLeakScanner.java

```diff
@@ -71,7 +71,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		    				"", // Other info
 		    				getSolution(), //Solution
 		    				getReference(), //References
-		    				xpbDirective,	// Evidence - Return the X-Powered-By Header info
+		    				"X-Powered-By: " + xpbDirective, // Evidence - Return the X-Powered-By Header info

```

Comment:
Thanks for tackling this. I'd suggest that we just highlight the header though, it could contain more than space (or who knows what other weird circumstance) so attempting to re-assemble it manually could fail to match properly.

"X-Powered-By" should probably be moved to a constant, then used in the getHeaders call and as the Evidence field.